sphinx/ext/autosummary/generate.py

```diff
@@ -79,6 +80,11 @@ def _simple_info(msg):
 def _simple_warn(msg):
     print('WARNING: ' + msg, file=sys.stderr)
 
+def _underline(title, line='='):
+    if '\n' in title:
+        raise ValueError('Can only underline single lines')
+    return title + '\n' + line*len(title)
```

Comment:
Yes, you can use east asian characters for python modules.
```
# main.py
import テスト

テスト.関数('Sphinx')
```
```
# テスト.py
def 関数(名前):
    print('Hello ' + 名前)
```

Okay, I will do it in another ticket.